toolkit/tools/pkg/imagecustomizerlib/customizefiles.go

```diff
@@ -17,8 +17,8 @@ const (
 	defaultFilePermissions = 0o755
 )
 
-func copyAdditionalFiles(baseConfigPath string, additionalFiles imagecustomizerapi.AdditionalFileList,
-	imageChroot *safechroot.Chroot,
+func CopyAdditionalFiles(baseConfigPath string, additionalFiles imagecustomizerapi.AdditionalFileList,
```

Comment:
You don't need to make a function public if you are accessing it from the same package/directory.